app/src/main/java/com/portfolio/tasky/BaseFragment.kt

```diff
@@ -0,0 +1,43 @@
+package com.portfolio.tasky
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+
+open class BaseFragment: Fragment() {
+    private var mActivity: BaseActivity? = null
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        mActivity = activity as BaseActivity
+    }
+
+    override fun onStart() {
+        super.onStart()
+        // Not implemented yet
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+    }
+
+    override fun onStop() {
+        super.onStop()
+        // Not implemented yet
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        // Not implemented yet
+    }
```

Comment:
Same here. No need to add this lines of code if they aren't being used in this pr

Reply:
Done